NCS.DSS.Outcomes/GetOutcomesByIdHttpTrigger/Function/GetOutcomesByIdHttpTrigger.cs

```diff
@@ -137,7 +137,9 @@ public async Task<IActionResult> RunAsync([HttpTrigger(AuthorizationLevel.Anonym
             _logger.LogInformation($"Function {nameof(GetOutcomesByIdHttpTrigger)} has finished invocation");
 
             if (outcomes == null)
+            {
                 return new NoContentResult();
+            }
 
             return new JsonResult(outcomes, new JsonSerializerOptions())
             {
```

Comment:
Need to return logging of retrieving the outcome is successful with its outcome ID